lib/ansible/modules/cloud/openstack/os_stack.py

```diff
@@ -34,6 +34,11 @@
       description:
         - Name of the stack that should be created, name could be char and digit, no space
       required: true
+    tag:
+      description:
+        - Tag for the stack that should be created, name could be char and digit, no space
+      required: false
+      default: None
```

Comment:
I think you'll need a "version_added" here.

Reply:
Added it ;)